lighthouserc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
Are we not able integrate this into wdio?

Reply:
I am not sure of this at this moment 🙂. I was bit confused by looking at documentation so added basic configurations to verify Lighthouse CI integration.